Removed the license statement from the readme file

ErikMogensen opened the pull request:
Since the license statement on the readme page is different from the license file, https://github.com/paolosalvatori/ServiceBusExplorer/blob/develop/LICENSE.txt this PR removes that part from the readme.

paolosalvatori reviewed (changes requested):
Let's do the opposite: let's remove the license from the readme page and let's update the license under license.txt with the following:

The MIT License (MIT)

Copyright (c) 2015 Microsoft Azure

Permission is hereby granted, free of charge, to any person obtaining a copy
of this software and associated documentation files (the "Software"), to deal
in the Software without restriction, including without limitation the rights
to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
copies of the Software, and to permit persons to whom the Software is
furnished to do so, subject to the following conditions:

The above copyright notice and this permission notice shall be included in all
copies or substantial portions of the Software.

THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
SOFTWARE.

SeanFeldman commented:
:x: [Build ServiceBusExplorer 5.1.0-ErikMogensen-updated-readme.38.build.864 failed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/39848557) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/7014465f81 by @ErikMogensen)

ErikMogensen commented:
That was a much shorter and better license statement.

SeanFeldman commented:
:x: [Build ServiceBusExplorer 5.1.0-ErikMogensen-updated-readme.40.build.866 failed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/39853892) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/a5a723503b by @ErikMogensen)

SeanFeldman commented:
@ErikMogensen im assuming you have no issues locally?

ErikMogensen commented:
@SeanFeldman, I have been running into this problem once in a while since I converted the project file for Common to SDK style. According to what I found on internet this issue may be caused by multiple reasons. One that was mentioned was mixing SDK style and classic project file in the same solution. This PR is based on such a mixed solution, but the latest PR #573 is using a solution which has only SDK style projects and that one hasn't had such problems .

It looks like there is some kind of timing issue since it sometimes works and sometimes not.

This PR should not affect the build since it changes just the readme.md, a text file and removed an empty line.

ErikMogensen commented:
If the problem would still occur the next step would be to add a `RuntimeIdentifiers` element according to [this ](https://stackoverflow.com/a/55406534/6850500)answer at StackOverflow.

ErikMogensen commented:
I suggest we merge this, but it's better that you do it, @SeanFeldman.

ErikMogensen commented:
I think the reason it built successfully the last time was that the SDK style projects had been merged into the develop branch then.

SeanFeldman commented:
:x: [Build ServiceBusExplorer 5.1.0-ErikMogensen-updated-readme.40.build.869 failed](https://ci.appveyor.com/project/seanfeldman/servicebusexplorer/builds/39887363) (commit https://github.com/paolosalvatori/ServiceBusExplorer/commit/27efde955d by @ErikMogensen)

SeanFeldman commented:
> I suggest we merge this, but it's better that you do it, @SeanFeldman.

@ErikMogensen I've triggered the build again and it's failing. Are you sure you want to merge this as-is?

ErikMogensen commented:
Hmm, I didn't expect that. I guess it is doing some strange merging since the changes were done on a branch which came from the master branch. I created #575 instead.
